content/operate/rc/supported-regions.md

```diff
@@ -0,0 +1,140 @@
+| `europe-west3`      | Frankfurt, Germany  | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |
+| `europe-west4`      | Netherlands         | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `europe-west6`      | Zurich, Switzerland | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+
+### Asia Pacific
+
+| Region ID              | Location             | Redis Cloud Pro | Redis Cloud Essentials |
+|------------------------|----------------------|-----------------|------------------------|
+| `asia-east1`           | Taiwan               | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `asia-east2`           | Hong Kong, China     | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `asia-northeast1`      | Tokyo, Japan         | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |
+| `asia-northeast2`      | Osaka, Japan         | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `asia-northeast3`      | Seoul, South Korea   | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `asia-south1`          | Mumbai, India        | <span title="Supported">&#x2705;</span> | <span title="Supported">&#x2705;</span> |
+| `asia-south2`          | Delhi, India         | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `asia-southeast1`      | Singapore            | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `asia-southeast2`      | Jakarta, Indonesia   | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `australia-southeast1` | Sydney, Australia    | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+| `australia-southeast2` | Melbourne, Australia | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+
+### Middle East
+
+| Region ID  | Location         | Redis Cloud Pro | Redis Cloud Essentials |
+|------------|------------------|-----------------|------------------------|
+| `me-west1` | Tel Aviv, Israel | <span title="Supported">&#x2705;</span> | <span title="Not supported">&#x274c;</span> |
+
+## Microsoft Azure
+
+Redis Cloud Essentials is available on the following Microsoft Azure regions:
+
+| Region ID | Location        |
+|-----------|-----------------|
+| `east-us` | Virginia, USA   |
+| `west-us` | California, USA |
+
+Redis Cloud Pro is available on Azure through Azure Cache for Redis Enterprise. See [Azure Cache for Redis pricing](https://azure.microsoft.com/en-us/pricing/details/cache/) to view the list of Azure regions that support Azure Cache for Redis Enterprise.
```

Comment:
not bolded enough. Maybe inside of a note?

Reply:
I put this part inside of a note, let me know what you think.